docs/source/release/v3.13.0/reflectometry.rst

```diff
@@ -24,8 +24,6 @@ Bug fixes
 Features Removed
 ################
 
-* Added deprecation notice to ISIS Reflectometry (Old) due to be removed in March 2019.
```

Comment:
Was this release note meant to be removed?

Reply:
It looks important.

Reply:
It was not meant to be removed.